var/spack/repos/builtin/packages/py-arcgis/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyArcgis(PythonPackage):
+    """ArcGIS API for Python."""
+
+    homepage = "https://developers.arcgis.com/python/"
+    pypi     = "arcgis/arcgis-1.8.4.tar.gz"
+
+    version('1.8.4', sha256='f1445dac25d3d4c03755d716c74a0930881c6be3cd36d22c6ff5ac754f9842d7')
```

Comment:
Theres a newer version.  Did you want to add 1.8.5?

Reply:
Nah, just needed the package for a hmwk assignment, no longer need it anymore